components/Aduan/Milestone.vue

```diff
@@ -1,322 +1,473 @@
 <template>
   <div class="w-full">
-    <div
-      v-for="(milestone, index) in dataMilestone"
-      :key="index"
-      class="flex mt-3"
-    >
-      <div class="flex flex-col items-center mr-4">
-        <div
-          class="flex items-center justify-center border dark:border-dark-emphasis-medium w-[26px] h-[26px] bg-gray-50 rounded-lg dark:bg-dark-emphasis-medium"
-        >
-          <BaseIconSvg
-            :icon="`/icon/${
-              getStatusTextAndIcon(
-                milestone.status_aduan,
-                milestone?.log_span_lapor?.status
-              ).icon
-            }`"
-            class="!shadow-lg !w-[14px] !h-[14px]"
-            :fill-color="
-              index > 0
-                ? '#868C89'
-                : `${
-                  getStatusTextAndIcon(
-                    milestone.status_aduan,
-                    milestone?.log_span_lapor?.status
-                  ).fillColor
-                }`
-            "
-          />
-        </div>
-        <div
-          v-if="index !== dataMilestone.length - 1"
-          class="w-px h-full bg-gray-300 dark:bg-dark-emphasis-medium"
-        />
-      </div>
-      <div class="w-full">
-        <CardMilestone>
-          <!-- text if id sp4n laporan is exist -->
-          <TextMilestone v-if="isSpanLapor(milestone)">
-            <span
-              :class="
+    <div v-for="(milestone, index) in dataMilestone" :key="index">
+      <div v-if="isStatusSpanLaporExist(milestone)" class="flex mt-3">
+        <div class="flex flex-col items-center mr-4">
+          <div
+            class="flex items-center justify-center border dark:border-dark-emphasis-medium w-[26px] h-[26px] bg-gray-50 rounded-lg dark:bg-dark-emphasis-medium"
+          >
+            <BaseIconSvg
+              :icon="`/icon/${
+                getStatusTextAndIcon(
+                  milestone.status_aduan,
+                  milestone?.log_span_lapor?.status
+                ).icon
+              }`"
+              class="!shadow-lg !w-[14px] !h-[14px]"
+              :fill-color="
                 index > 0
-                  ? 'text-gray-400 dark:text-dark-text-low dark:text-opacity-60'
-                  : 'text-gray-600 dark:text-dark-text-low '
+                  ? '#868C89'
+                  : `${
+                    getStatusTextAndIcon(
+                      milestone.status_aduan,
+                      milestone?.log_span_lapor?.status
+                    ).fillColor
+                  }`
               "
-            >
-              ID Tracking SP4N LAPOR
-            </span>
-            <div>
+            />
+          </div>
+          <div
+            v-if="index !== dataMilestone.length - 1"
+            class="w-px h-full bg-gray-300 dark:bg-dark-emphasis-medium"
+          />
```

Comment:
Hello @rayfajars, is width in the class is typo?

Reply:
in [documentation](https://tailwindcss.com/docs/width) tailwind w-px is width 1 px 

![image](https://github.com/jabardigitalservice/super-app-web/assets/58242304/06282a8c-9b5b-433f-958c-67484020dd7e)